app/services/campaign_page.rb

```diff
@@ -20,6 +22,7 @@ def initialize(utm_content_code, criteria)
   end
 
   def banner_title(name, subject = nil)
```

Comment:
I know its not in this PR, but if you don't pass 'subject' here then don't you get an extra space in the text 'find the right  job for you'...?